src/content/docs/en/guides/integrations-guide/partytown.mdx

```diff
@@ -149,6 +151,38 @@ export default defineConfig({
 });
 ```
 
+### config.resolveUrl
+
+Some third-party scripts may require proxying through `config.resolveUrl()`, which runs inside the service worker. [Read more in Partytown's documentation.](https://partytown.qwik.dev/forwarding-events)
+
+However since `config` is serialized (even functions), it has limitations:
+
+- It cannot reference anything outside of the function scope
+- It can only use JavaScript
+
+For advanced use cases, you may need to pass data to this function while initializing Partytown. To do so, you can set `resolveUrl()` on `window.partytown` instead of the integration config:
```

Comment:
```suggestion
In some advanced use cases, you may need to pass data to this function while initializing Partytown. To do so, you can set `resolveUrl()` on `window.partytown` instead of the integration config:
```
Great intro here!